JSDemos/Demos/DataGrid/CustomSummaries/description.md

```diff
@@ -1 +1,15 @@
-The DataGrid support custom summary calculation. In&nbsp;this example, summaries are only calculated against selected grid rows. This behavior is&nbsp;implemented using the [calculateCustomSummary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/#calculateCustomSummary) and [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) properties.
\ No newline at end of file
+DataGrid can calculate custom summaries on the client or server side. In this demo, we have implemented the client-side logic. The instructions below explain how to do this in your code:
+
+1. Make sure that the **remoteOperations**.[summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/remoteOperations/#summary), **remoteOperations**.[groupPaging](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/remoteOperations/#groupPaging), or [remoteOperations](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/remoteOperations/) property is not set or set to **false**.
+
+1. Add a summary configuration object to the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[groupItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/) or [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array.
+
+1. Set the object's [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType) to *"custom"*.
+
+1. Specify the summary's [name](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#name). It will be used to identify the summary item within the **calculateCustomSummary** function in the next step.
+
+1. Calculate the resulting value in the [calculateCustomSummary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/#calculateCustomSummary) function. Implement the necessary calculation stages. See the function's description for details.
+
+The custom summary in this example calculates the sum of sale amounts for selected rows. To recalculate the resulting value on selection change, the code calls the [refresh()](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#refresh) method in the [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) handler.
```

Comment:
resulting value on selection change?
what are you trying to say?

Reply:
![image](https://user-images.githubusercontent.com/66619750/110909032-a972d780-8320-11eb-8e1f-f735c8166503.png)
Each time a user selects/deselects an item, the summary ("the resulting value") is recalculated (please see the image).